internal/pkg/agent/cmd/container.go

```diff
@@ -92,11 +92,13 @@ The following actions are possible and grouped based on the actions.
   FLEET_SERVER_ELASTICSEARCH_CA_TRUSTED_FINGERPRINT - The sha-256 fingerprint value of the certificate authority to trust
   FLEET_SERVER_ELASTICSEARCH_INSECURE - disables cert validation for communication with Elasticsearch
   FLEET_SERVER_SERVICE_TOKEN - service token to use for communication with elasticsearch
+  FLEET_SERVER_SERVICE_TOKEN_FILE - to service token file to use for communication with elasticsearch
```

Comment:
Would you be okay with changing the name to `FLEET_SERVER_SERVICE_TOKEN_PATH`? I am thinking that `_PATH` would be a better name as it would name it as a path to a file.

Not a hard opinion on it, just kinda think I like that naming better.